web/src/components/map/index.js

```diff
@@ -29,11 +30,12 @@ import { Grid } from "../ui";
  */
 export const Map = () => {
   const location = useLocation(); // location for the url
-  const initialDate = moment(location?.state?.date) || moment(); // Date either from upload or current day
+  const initialDate = moment(location?.state?.date); // Date either from upload or current day (moment(undefined) gives now)
   const [mapDate, setMapDate] = useState(initialDate);
   const [formattedDate, setFormattedDate] = useState(
     initialDate.format("YYYY-MM-DD")
   );
+  const [setOfDates, setSetOfDates] = useState(() => new Set()); // Lazy initial render
```

Comment:
My personal choice for sets is to put `unique` in the name. Otherwise, we can end up in situations where we're saying `setSet...`

```suggestion
  const [allDatesUnique, setAllDatesUnique] = useState(() => new Set()); // Lazy initial render
```

Reply:
I was wondering if there was a better convention. Will change in index.js, menu.js, and menu.test.js.